Tests/PackageLoadingTests/PD4_2LoadingTests.swift

```diff
@@ -198,6 +201,16 @@ class PackageDescription4_2LoadingTests: XCTestCase {
             XCTAssertEqual(deps["/foo7"], PackageDependencyDescription(url: "/foo7", requirement: .branch("master")))
             XCTAssertEqual(deps["/foo8"], PackageDependencyDescription(url: "/foo8", requirement: .upToNextMinor(from: "1.3.4")))
             XCTAssertEqual(deps["/foo9"], PackageDependencyDescription(url: "/foo9", requirement: .upToNextMajor(from: "1.3.4")))
+        
+            let homeDir = localFileSystem.homeDirectory!.asString
```

Comment:
I thought it's of no matter because it is test.

I'm going to use `XCTFail` instead.

```swift
guard let homeDir = localFileSystem.homeDirectory?.asString else {
    return XCTFail("Could not detect home directory")
}
```